### PR TITLE
feat: Add issue templates for service catalog

### LINKED ISSUE #72: Create issue templates
Since the documentation (in #73) mentions creating issues, we should have issue templates.

### PR DESCRIPTION
**Pull request by mimotej**
Resolves #72 
For now I have created following issue templates:
- [X] Bug report
- [X] Feature request

If you think there should be any other templates present or you want some changes in current please let me know

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    required: true
+- type: textarea
+  id: links
+  attributes:
+    label: Relevant links
+    description: |
+      Add any links to resources, issues, etc. that you feel are
+      relevant to this story.
+  validations:
+    required: true
```
Comment (tumido) [inline .github/ISSUE_TEMPLATE/feature_request.yaml:29]: Why are links required here?

**Review by tumido — COMMENTED** [review]
/lgtm
/approve

File: .github/ISSUE_TEMPLATE/feature_request.yaml (hunk 1 above)
Comment (mimotej) [reply .github/ISSUE_TEMPLATE/feature_request.yaml]: Leftover validation from different issue, removed it :+1:

**Review by tumido — APPROVED** [review]
/lgtm
/approve

**Pull request merged**